README.md

```diff
@@ -19,17 +19,27 @@ None of them is dealing with jira.
 - [ ] optional: insert build types (e.g. beta or production)
 
 ## Requirements
-- python 2.x
-- [jira-python](https://github.com/pycontribs/jira)
-    - install via `pip install jira`
+- Python 2.x
+  - `pip` for installing dependencies
 - jira admin rights - to create fix versions
 
-[1]: https://keepachangelog.com/en/1.0.0/
+## Setup
+
+On Python 3.3 and up, start the [virtual environment][1] in order to keep everything self-contained:
```

Comment:
i think the script is not compatible with python3 yet, will add a todo to actually convert it to python3
also it's not required as setup here, more like a tip.